brands/tourism/hotel.json

```diff
@@ -635,6 +751,15 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|The Ritz-Carlton": {
```

Comment:
Add matching names for “Ritz-Carlton”, “Ritz”, and “The Ritz”.

Reply:
Fixed